### PR TITLE
Refactors project to use Svelte

### PR DESCRIPTION
**Pull request by livtanong**
Implementation of: #13 

As far as I know, this is feature complete.

### Changes From Current
1. Appearance
2. Analysis does not need a button to trigger. Starts immediately after upload.
3. Each phenotype header shows how many variants are pathogenic
4. Rendering of pathogenicity of a variant is now an explicit label, rather than coloring the entire line red

### Additions
1. Genotype class - An abstraction so that the developer does not have to worry about flipping the order of the nucleotides
2. GeneDataParser class - For encapsulating parsing methods.
3. GeneVariant class - So we can have nice methods

### To do
1. More user-friendly error handling and reporting now that we have access to Svelte-powered UI. I have put this off to minimize the scope of this PR.
2. Probably drag and drop for file upload.
3. Delete the `public` directory

### Notes
1. Not actually sure if gh-deploy will work properly
2. I want to do so much more with the UI, but I have tried to keep it as close to status quo as possible, to limit the scope to a *refactor* rather than a redesign.

### CODE REVIEW

File: static/mps/mps-data.json [hunk 1]
```diff
@@ -161,41 +161,6 @@
...
-    "rs2736990": {
-        "phenotype": "NAD(P)H Metabolism",
-        "pathogenic": ["CC", "CT"],
-        "gene": "SNCA"
-    },
-    "rs356219": {
-        "phenotype": "NAD(P)H Metabolism",
-        "pathogenic": ["GG", "AG"],
-        "gene": "LOC105377329"
-    },
```
Comment (d6e) [inline static/mps/mps-data.json:198]: Why were these lines deleted?

Comment (livtanong) [reply static/mps/mps-data.json:198]: Oh those were from https://github.com/snpeek/snpeek.github.io/pull/18
Because it was already approved, I figured it would be merged eventually, so I didn't bother branching it off to a common parent.

Comment (d6e) [reply static/mps/mps-data.json:198]: Ah that makes sense

**Comment by d6e** [comment]
This seems good, if you can just bring it up-to-date with the latest main, I'll fix up the CI part and merge it.

**Comment by livtanong** [comment]
@d6e done! :D

**Comment by d6e** [comment]
Alright looks good! Thank you so much!

**Review by d6e — APPROVED** [review]
